src/install/haproxy/find-data.mdx

```diff
@@ -0,0 +1,16 @@
+---
+componentType: default
+headingText: Find and use data
+---
+
+Data from this integration can be found by going to: **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Infrastructure > Third-party services > HAProxy**. 
+
+Metrics are attached to the following [event types](/docs/using-new-relic/data/understand-data/new-relic-data-types#events-new-relic):
```

Comment:
Might clarify that they need to search for these event types to find the metrics. (i think that's how it works but I can't test it since I don't have this installed.)